server/src/main/java/io/druid/metadata/IndexerSQLMetadataStorageCoordinator.java

```diff
@@ -461,6 +475,7 @@ public SegmentIdentifier inTransaction(Handle handle, TransactionStatus transact
                   .bind("end", interval.getEnd().toString())
                   .bind("sequence_name", sequenceName)
                   .bind("sequence_prev_id", previousSegmentIdNotNull)
+                  .bind("sequence_name_prev_id_sha1", sequenceNamePrevIdSha1)
```

Comment:
can we call this "sqeuence_name_and_prev_id_sha1" as its not just prev_id hashed in there.